about-this-guidebook/README.md

```diff
@@ -18,10 +18,10 @@ When making a contribution to the guidebook:
 
 3. Take 10 minutes to review:
 
-   - The [writing style](writing-style-guide.md) for guidebook content (to maintain consistency).
-   - How to do [Markdown](markdown-for-guidebook.md).
-   - How to [edit content using Git](editing-the-guidebook.md)
-   - Who [approves pull requests](guidebook-governance.md) (PRs) when you make changes
+   1. The [writing style](writing-style-guide.md) for guidebook content (to maintain consistency).
+   2. How to do [Markdown](markdown-for-guidebook.md).
+   3. How to [edit content using Git](editing-the-guidebook.md)
+   4. Who [approves pull requests](guidebook-governance.md) (PRs) when you make changes
 
```

Comment:
I don't see a line break there?

Reply:
Ya seems like our list indenting is not working as expected. I spend some time debugging yesterday but don't have a resolution. I do have an idea of what is happening but need to confirm. It is some of checks that are blocking it I believe.

Reply:
I believe the issue that to have nested indentation in markdown we need to use 4 spaces but our checks were replacing 4 spaces with 3 spaces. I couldn't track down where we set that but I believe it just what the 'prettier' check for forcing on us. @grugnog need your input here.

Now after setting 4 spaces with a settings file (`.prettierrc`) it went through and updated all our files "fixing" the indents. That is good as now other places where there was a indented list will appear properly. Compare https://guidebook.civicactions.com/en/latest/practice-areas/project-management/mods-and-extensions/#common-steps-for-a-mod-or-extension with https://civicactions-handbook--1042.org.readthedocs.build/en/1042/practice-areas/project-management/mods-and-extensions/#common-steps-for-a-mod-or-extension. But this is bad because of the amount of changes that just occured.

I have struggled with indent issue for a while, last example was https://github.com/CivicActions/guidebook/pull/870. Since it is common place for use of indented lists I think we want this fix.

For now I made this a draft MR so we review.

Reply:
FYI I removed my changes that caused all the files to be updated.

Reply:
@ContentBrewmaster I will merge this and then in a separate pull request fix the indentation.